astro/src/content/docs/extend/examples/api-gateways/index.mdx

```diff
@@ -5,6 +5,7 @@ navcategory: developer
 section: extend
 subcategory: examples
 tertcategory: api gateways
+topOfNav: true
```

Comment:
The navigation reads only API Gateways, not API Gateways Overview

Reply:
I'm not sure I understand.